Plugins/Sycl/include/Acts/Plugins/Sycl/Seeding/detail/Types.h

```diff
@@ -0,0 +1,64 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+#include <cstddef>
+#include <cstdint>
+
+namespace Acts::Sycl::detail {
+struct deviceSpacePoint {
+  float x;
+  float y;
+  float z;
+  float r;
+  float varR;
+  float varZ;
+};
+
+// Parameters required to calculate circle with linear equation.
+struct deviceLinEqCircle {
```

Comment:
Wow... I'm pretty baffled by GitHub right now... Is it not possible to un-resolve a discussion? :confused: In any case... Let's keep the discussion from https://github.com/acts-project/acts/pull/416/files#r481815029 going here then.

The capitalisation of these type names will need to change, according to the Acts coding rules.

Reply:
Oh yes, you're right. I was just reading fast, and thought you're talking about starting namespaces with capital letters. Sorry, I'll be more careful next time.